src/components/common/RegAddEditOrgPerson.vue

```diff
@@ -180,11 +180,8 @@
             </template>
 
             <!-- Roles -->
-            <article class="mt-8">
-              <label>{{ getPeopleAndRolesResource.rolesTitle || 'Roles' }}</label>
-              <div v-if="getPeopleAndRolesResource.rolesSubtitle">
-                {{ getPeopleAndRolesResource.rolesSubtitle }}
-              </div>
+            <article class="mt-8" v-if="showRoles">
+              <label>Roles</label>
```

Comment:
As you've seen in design discussions and uxpin changes, Relationships is no longer prompted here, so the former article title is fine.

This is also why I removed the additional roles (relationships) below.